extensions/wikia/FounderProgressBar/FounderProgressBarController.class.php

```diff
@@ -37,7 +37,6 @@ class FounderProgressBarController extends WikiaController {
 		'FT_RECENTCHANGES_VISIT' => 230,
 		'FT_WORDMARK_EDIT' => 240,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "FT_WORDMARK_EDIT" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)